READMEs/c2d-flow.md

```diff
@@ -41,15 +41,53 @@ This example uses c2d to create a regression model. In order to visualise it or
 In your project folder, in this case my_project from `Setup : Install the library` in First Steps, run the following command:
 
 ```console
-pip install numpy matplotlib
+#grab repo
```

Comment:
@mariacarmina Can you please revert these changes?  It looks like they were accidentally added back in during a merge.  They violate the DRY conventions added by @md0x README cleanup PR #696.

https://github.com/oceanprotocol/ocean.py/pull/698/files#diff-b30c139cd97de56d4e61b367afaa7c0477f931bd2f7bc1d681efe72765655207R44-R73
https://github.com/oceanprotocol/ocean.py/pull/698/files#diff-b30c139cd97de56d4e61b367afaa7c0477f931bd2f7bc1d681efe72765655207R81-R90